modules/openapi-generator/src/main/resources/kotlin-client/libraries/jvm-volley/request/IRequestFactory.mustache

```diff
@@ -0,0 +1,64 @@
+package {{packageName}}.request
+
+import com.android.volley.Request
+import com.android.volley.Response
+import java.io.UnsupportedEncodingException
+import java.lang.reflect.Type
+import java.net.URLEncoder
+import java.text.ParseException
+import java.text.SimpleDateFormat
+import java.util.*
+import java.time.format.DateTimeFormatter
+import java.time.OffsetDateTime
+import java.time.LocalDate
+
+
+interface IRequestFactory {
+
+    companion object {
+        /**
+         * ISO 8601 date time format.
+         * @see https://en.wikipedia.org/wiki/ISO_8601
+         */
+        fun formatDateTime(datetime: OffsetDateTime) = DateTimeFormatter.ISO_INSTANT.format(datetime)
```

Comment:
@alisters Here you should use DateTimeFormatter.ISO_OFFSET_DATE_TIME

Reply:
@4brunu - is there a reason for preferring ISO_OFFSET_DATE_TIME over ISO_INSTANT ? The link you supplied me https://swagger.io/docs/specification/data-models/data-types/, and it's relevant section 

date-time – the date-time notation as defined by RFC 3339, section 5.6, for example, **_2017-07-21T17:32:28Z_**  - would seem to suggest ISO_INSTANT ? 

I can't give you a solid reason for that, it's been a while since i had to think through this in our stack i just have a nagging feeling we landed on UTC representation for a reason. If you have a strong preference for ISO_OFFSET_DATE_TIME  i can try and dig up relevant design discussion notes. I don't think it will make a difference for this location, as mentioned we don't use the format for our date fields, i just think we might have chosen it to line up with how we serialize dates in other sections of our stack.

Reply:
You are right, let's keep it as is. I suggested this because it's what the other kotlin generators use, but I think you are right. 👍